src/gui/tray/usermodel.cpp

```diff
@@ -697,6 +702,32 @@ bool User::hasActivities() const
     return _account->account()->capabilities().hasActivities();
 }
 
+QColor User::headerColor() const
+{
+    const auto serverColor = _account->account()->capabilities().serverColor();
+    return serverColor.isValid() ? serverColor : Theme::defaultColor();
+}
+
+QColor User::headerTextColor() const
+{
+    return _account->account()->capabilities().serverTextColor();
+}
+
+QColor User::accentColor() const
+{
+    // This will need adjusting when dark theme is a thing
+    const auto serverColor = _account->account()->capabilities().serverColor();
+
+    if(!serverColor.isValid()) {
+        return QColor();
+    }
+
+    auto darknessAdjustment = (int)((1 - Theme::getColorDarkness(serverColor)) * 8);
+    darknessAdjustment *= darknessAdjustment; // Square the value to pronounce the darkness more in lighter colours
+    const auto adjusted = Theme::isDarkColor(serverColor) ? serverColor : serverColor.darker(125 + darknessAdjustment);
```

Comment:
@claucambra What do these numbers (125, 8) mean? Could you move them into consts (ok to place within this scrope) with nice names?

Reply:
Sure, added descriptive names for them and set as consts